components/Navbar.tsx

```diff
@@ -184,28 +189,37 @@ const routes: NavSection[] = [
 
 export const Navbar: React.FC = () => {
   const [isMobileMenuOpen, setIsMobileMenuOpen] = useState(false)
+  const [openSubmenus, setOpenSubmenus] = useState<string[]>([]); // Array to track open submenus
 
   const toggleMobileMenu = () => {
     setIsMobileMenuOpen(!isMobileMenuOpen)
   }
 
+  const toggleSubmenu = (sectionName: string) => {
+    if (openSubmenus.includes(sectionName)) {
+      setOpenSubmenus(openSubmenus.filter(name => name !== sectionName));
+    } else {
+      setOpenSubmenus([...openSubmenus, sectionName]);
+    }
+  };
+
   return (
     <header className={`sticky top-0 z-50`}>
       <nav className="bg-blue-navbar flex px-8 justify-between">
         {/* CCV Logo */}
-        <div className="flex items-center py-4 lg:py-8">
+        <div className="flex items-center py-4 lg:py-8 px-12">
           <Link href={"/"}>
             <CCVLogo width={120}/>
           </Link>
         </div>
 
         {/* Navigation Menu for Desktop */}
-        <NavigationMenu.Root className="hidden lg:flex relative z-10 w-screen justify-between items-stretch">
+        <NavigationMenu.Root className="xl:flex hidden relative z-10 w-screen justify-between items-stretch">
           <NavigationMenu.List className="m-0 flex list-none rounded-md h-full items-center">
             {routes.map((section) =>
               <NavigationMenu.Item key={section.name}>
-                <NavigationMenu.Trigger className="group inline-flex h-9 items-center justify-center gap-2 px-4 text-white text-2xl transition-colors hover:text-sunglow-400 focus:outline-none disabled:pointer-events-none disabled:opacity-50">
-                  {section.name}
+                <NavigationMenu.Trigger className="group inline-flex h-9 items-center justify-center gap-2 px-6 text-white text-xl font-semibold transition-colors hover:text-sunglow-400 focus:outline-none disabled:pointer-events-none disabled:opacity-50">
+                  {section.name.toUpperCase()}
```

Comment:
I don't really like the uppercasing 😢. Maybe go back to regular casing?

Reply:
But if you like the capital case... we could... i guess 😅

Reply:
i want to like it bc it's like brown websites... but i havent been able to get rid of the horrible feeling in my stomach everytime i lay  my eyes on it :D i'm gonna change